Changelog.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+**1.4.2**
```

Comment:
I tag after the merge request directly on master with commit type release :wink: